src/ESPEasyRules.ino

```diff
@@ -889,7 +889,7 @@ bool conditionMatchExtended(String& check) {
     condOr  = check.indexOf(F(" or "));
 
     if ((condAnd > 0) || (condOr > 0)) {                             // we got AND/OR
-      if ((condAnd > 0) && (((condOr < 0) && (condOr < condAnd)) ||
+      if ((condAnd > 0) && (((condOr < 0) /*&& (condOr < condAnd)*/) ||
```

Comment:
you can just remove this part

Reply:
Yep, but I left it in there as a reference (for you I guess :) )
It is redundant code, as PVS-Studio also noticed.